knowledge-base/basic-authentication-with-fiddler-everywhere.md

```diff
@@ -18,55 +18,56 @@ res_type: kb
 
 #### Description
 
-Fiddler Everywhere certificate allows the inspection and debugging of network traffic. The __Composer__ feature in Fiddler Everywhere enables you to manually build and send HTTP or HTTPS requests. Click [here]({%slug composer-tab%}) to know more about Fiddler Everywhere Composer.
-
 In the context of an HTTP transaction, __Basic Access Authentication__ is a method for an HTTP user agent (for example, a web browser) to provide a user name and password when making a request. In __Basic HTTP Authentication__, a request contains a header field in the form of __`Authorization: Basic <base64 string>`__, where credentials are the __Base64__ encoding of `username` and `password` joined by a single colon(`:`).
 
-Now, let's see how you can create a Composer request for APIs that require __Authentication__:
-
->tip In the steps shown below, you can use [httpbin.org](https://httpbin.org/) for HTTP request and response service.
+In this article, we list the steps to [create a request for APIs that require __Authentication__](#create-a-basic-authentication-request) by using the Fiddler Everywhere [Composer]({%slug composer-tab%}) and provide a [complete example](#basic-authentication-request-example) of the procedure.
 
-## Step 1
+## Create a Basic Authentication Request
```

Comment:
I suggest extracting the required steps in one section and then adding a second section with the example for a better scannability of the content.